lib/get-values-from-ci.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash')
+
+const integrationImports = [
+  require('./ci-integrations/travis-ci'),
+  require('./ci-integrations/circle-ci'),
+  {test: _.stubTrue, getValues: _.constant(null)}
+]
+
+const integrations = _.map(integrationImports, function (integration) {
+  return [_.get(integration, 'test', _.stubFalse), _.get(integration, 'getValues', _.noop)]
```

Comment:
I wouldn't use defaults here. If a ci-integration doesn't export these functions I'd prefer this to fail on us early.